MaxText/pyconfig.py

```diff
@@ -191,6 +191,12 @@ def validate_keys(keys):
 
   if keys["use_multimodal"]:
     validate_multimodal_model_name(keys["model_name"])
+    if keys["use_sft"]:
+      assert keys[
+          "sft_train_on_completion_only"
+      ], "In multimodal SFT (use_multimodal=True, use_sft=True), sft_train_on_completion_only must be set to True"
+      # TODO(aireenmei, hengtaoguo): support packing
```

Comment:
For packing, does it concatenates string until it reaches the maximum input/seq length?

Reply:
Yes, this is true for text. We will add packing support for multimodal later, which will include implementations for passing multiple images of different sizes in the data pipeline, and text image fusion on embedding outputs.